app/controllers/internal/broadcasts_controller.rb

```diff
@@ -21,7 +29,11 @@ def edit
   end
 
   def index
-    @broadcasts = Broadcast.all
+    @broadcasts = if params[:type_of]
+                    Broadcast.where(type_of: params[:type_of].capitalize)
```

Comment:
should these be ordered alphabetically or by descending creation date?

Reply:
Good call! We don't have `created_at` on these (which is odd, and which I didn't notice til just now!). But for the time being, I am going with ordering alphabetically 👍

Reply:
Ooh! that's weird. We should definitely send a PR to add those fields ahhaha. I'm suprised Rails works without the two timestamps at all.

TIL!

Reply:
https://github.com/thepracticaldev/dev.to/pull/8181